web/vibe/web/rest.d

```diff
@@ -2107,10 +2107,26 @@ private HTTPClientResponse request(URL base_url,
 
 	if (!isSuccessCode(cast(HTTPStatus)client_res.statusCode))
 	{
-		Json msg = Json(["statusMessage": Json(client_res.statusPhrase)]);
-		if (client_res.contentType.length)
-			if (client_res.contentType.splitter(";").front.strip.sicmp("application/json") == 0)
+		import std.base64;
+
+		Json msg;
+		if (client_res.contentType.length) {
+			const ctype = client_res.contentType.splitter(";").front.strip;
+			if (ctype.sicmp("application/json") == 0)
 				msg = client_res.readJson();
```

Comment:
Will this case contain `statusMessage`? Should we have `"statusMessage": Json(client_res.statusPhrase)` here too?

Reply:
Yeah, `RestException` already has `statusCode`.

Just pointing out that all cases except `application/json` have `statusMessage` field set.